sdk/tables/azure-data-tables/azure/data/tables/_models.py

```diff
@@ -448,8 +417,8 @@ def from_string(
         :param str permission: Specify permissions in
             the string with the first letter of the word.
         :keyword callable cls: A custom type or function that will be passed the direct response
```

Comment:
Could we remove this line? It's causing some weirdness in APIview

Reply:
I searched cls docstring across services, don't see any service add line for cls. I think we can remove this line, will update in next commit.